### PR TITLE
[major] Support Strimzi as default Kafka provider

### PR DESCRIPTION
**Pull request by andrercm**
- Added support for Kafka providers to be selected while installing IoT in MAS cli (amq, strimzi, event streams and msk)
- Added BYO support if no Kafka providers are needed
- Changes are supporting both interactive or silent install modes.

### CODE REVIEW

File: docs/commands/install.md [hunk 1]
```diff
@@ -49,6 +49,33 @@ Usage
 ### IBM Cloud Pak for Data (Required when installing Predict or Assist):
 - `--cp4d-version CP4D_VERSION`                                  Product version of CP4D to use
 
+### Kafka - Common Arguments (Optional, required to install Maximo IoT):
+- `--kafka-provider KAFKA_PROVIDER`                             Required. Set Kafka provider. Supported options are `redhat` (Red Hat AMQ Streams), `strimzi` and `ibm` (IBM Event Streams)
```
Comment (whitfiea) [inline docs/commands/install.md]: not aws?

File: image/cli/mascli/functions/install [hunk 2]
```diff
@@ -51,6 +51,18 @@ Maximo Application Suite Application Selection (Optional):
 IBM Cloud Pak for Data (Required when installing Predict or Assist):
       --cp4d-version ${COLOR_YELLOW}CP4D_VERSION${TEXT_RESET}                                  Product version of CP4D to use
 
+Kafka (Required to install Maximo IoT thus if not set, a default Kafka provider will be installed):
+      --kafka-provider ${COLOR_YELLOW}KAFKA_PROVIDER${TEXT_RESET}                              Set Kafka provider. Supported options are 'redhat' (Red Hat AMQ Streams), 'strimzi' and 'ibm' (IBM Cloud Event Streams)
```
Comment (whitfiea) [inline image/cli/mascli/functions/install]: or AWS?

File: image/cli/mascli/functions/pipeline_show_config [hunk 3]
```diff
@@ -51,7 +51,7 @@ function pipeline_show_config() {
   else echo_reset_dim " - Monitor ............... ${COLOR_MAGENTA}${MAS_APP_CHANNEL_MONITOR}"
   fi
 
-  if [[ "${MAS_APP_CHANNEL_MANAGE}" == '' ]]
+   if [[ "${MAS_APP_CHANNEL_MANAGE}" == '' ]]
```
Comment (whitfiea) [inline image/cli/mascli/functions/pipeline_show_config]: I don't think this indent is needed?

File: image/cli/mascli/functions/pipeline_show_config [hunk 4]
```diff
@@ -138,6 +138,39 @@ function pipeline_show_config() {
...
+  fi
+  if [[ "$KAFKA_ACTION_SYSTEM" == "install" && "$KAFKA_PROVIDER" == "ibm" ]]; then
+    reset_colors
+    echo "${TEXT_DIM}"
+    echo_h4 "Kafka - IBM Cloud Event Streams" "    "
+    echo_reset_dim "Resource group ............ ${COLOR_MAGENTA}${EVENTSTREAMS_RESOURCEGROUP}"
+    echo_reset_dim "Instance name ............. ${COLOR_MAGENTA}${EVENTSTREAMS_NAME}"
+    echo_reset_dim "Instance location ......... ${COLOR_MAGENTA}${EVENTSTREAMS_LOCATION}"
+  fi
+    if [[ "$KAFKA_ACTION_SYSTEM" == "install" && "$KAFKA_PROVIDER" == "aws" ]]; then
```
Comment (whitfiea) [inline image/cli/mascli/functions/pipeline_show_config]: indent wrong here?

**Review by whitfiea — CHANGES_REQUESTED** [review]

**Review by whitfiea — APPROVED** [review]

**Review by durera — APPROVED** [review]

**Pull request merged**